ramls/reporting_code.raml

```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
-title: "Sample-Module"
-baseUri: http://github.com/org/folio/sample-module
+title: "Orders Storage"
+baseUri: http://github.com/org/folio/mod-orders-storage
 version: v1
 
 documentation:
```

Comment:
please update the "title" field in the documentation section for consistency with the other APIs